api/services/elastigroup/azure/spotVms/paths/groupsVm.yaml

```diff
@@ -13,7 +13,7 @@ post:
       application/json:
         schema:
           allOf:
-          - $ref: "../schemas/elastigroupVm.yaml"
```

Comment:
@vlad666spot this was addressed in the previous PR which was approved, please make sure to align this branch with master